src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CreateProbationBookingService.kt

```diff
@@ -52,6 +52,8 @@ class CreateProbationBookingService(
       probationMeetingType = request.probationMeetingType!!.name,
       comments = request.comments,
       createdBy = createdBy,
+      notesForStaff = request.notesForStaff?.takeIf { createdBy is ExternalUser },
```

Comment:
We will only use private notes when requested by an external user.

Reply:
ditto